test/e2e/space_autocompletion_test.go

```diff
@@ -100,9 +86,8 @@ func TestAutomaticClusterAssignment(t *testing.T) {
 
 	t.Run("set low capacity threshold and expect that space will have default tier, but won't have target cluster so it won't be provisioned", func(t *testing.T) {
 		// given
-		hostAwait.UpdateToolchainConfig(t,
-			testconfig.CapacityThresholds().ResourceCapacityThreshold(1),
-		)
+		spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait1.ClusterName, testSpc.MaxMemoryUtilizationPercent(1), testSpc.Enabled(true))
+		spaceprovisionerconfig.UpdateForCluster(t, hostAwait.Awaitility, memberAwait2.ClusterName, testSpc.MaxMemoryUtilizationPercent(1), testSpc.Enabled(true))
```

Comment:
one thing that I noticed is that you explicitly define  `testSpc.Enabled(true)` everywhere in `space_autocompletion_test.go` even though it should be already set in https://github.com/codeready-toolchain/toolchain-e2e/blob/66d668ac8863788c0d240bfe1dd8de43013ca77d/deploy/host-operator/default-spaceprovisionerconfig.yaml#L13

For example the  `usersignup_test.go` doesn't have it - you don't explicitly define it there

Reply:
it's because the tests reuse the same client and disable/enable the SPCs back and forth. So I'm being explicit about the enabled states so that there is no confusion about the state of the SPCs.

Reply:
I know it should not be needed now that we have proper cleanup of the SPCs but I still find it provides for better clarity.